circle.yml

```diff
@@ -20,21 +20,21 @@ dependencies:
 
   override:
     - yarn
-    - npm rebuild
+    - yarn rebuild
```

Comment:
The yarn docs [say](https://yarnpkg.com/en/docs/migrating-from-npm) that this should be `yarn install --force`.